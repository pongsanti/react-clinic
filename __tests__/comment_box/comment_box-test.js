import React from 'react';
import CommentBox from '../../src/comment_box/comment_box.jsx';
import renderer from 'react-test-renderer';

describe('CommentBox', () => {
  it('is a div with text', () => {
    const component = renderer.create(
      <CommentBox />
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

  });

});