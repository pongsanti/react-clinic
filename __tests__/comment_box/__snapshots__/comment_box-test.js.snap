exports[`CommentBox is a div with text 1`] = `
<div
  className="commentBox">
  Hello, world!, I am a CommentBox.
</div>
`;
